cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,249 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-11944` - Removed non-configurable properties from CSD/Ambari
+
+- :cask-issue:`CDAP-11948` - Fix a bug where committed data could be removed during HBase table flush or compaction.
+
+- :cask-issue:`CDAP-11955` - Fixed a bug that sometimes wrong user is used in explore, which results in the failure of deleting namespace.
+
+- :cask-issue:`CDAP-11985` - Fixed a bug that ugi provider returns the old and incorrect ugi information.
+
+- :cask-issue:`CDAP-12054` - Fixed PartitionedFileSet to work with CombineFileInputFormat, as input to a batch job.
+
+- :cask-issue:`CDAP-12122` - Fixed a bug in the pipeline planner that caused some pipelines to fail to deploy with a NoSuchElementException
+
+- :cask-issue:`CDAP-12125` - Fixed a bug in mapreduce pipeline timing metrics, where time for a stage could include time spent in other stages.
+
+- :cask-issue:`CDAP-12130` - Fixed an issue that will enable send-to-directive working on derived columns in data prep.
+
+- :cask-issue:`CDAP-12161` - Fixed a bug in StructuredRecord where a union of null and at least two other types could not be set to a null value.
+
+- :cask-issue:`CDAP-12170` - Fixed a bug where committed files of a PartitionedFileSet could be removed during transaction rollback in the case PartitionOutput#addPartition was called for a partition that already e
+
+- :cask-issue:`CDAP-12193` - Fixed a bug in some mapreduce pipelines that could cause duplicate reads if sources are not properly merged into the same mapreduce.
+
+- :cask-issue:`CDAP-12194` - Removed dataset usage in the Hive source and sink, which allows it to work in Spark and fixes a race condition that could cause pipelines to fail with a transaction conflict exception.
+
+- :cask-issue:`CDAP-12199` - Fixed a bug that made local datasets inaccessible in a Workflow's initialize and destroy methods.
+
+- :cask-issue:`CDAP-12253` - Fixed a bug that the file batch source always uses a default schema instead of the actual output schema.
+
+- :cask-issue:`CDAP-12269` - Fixed a bug that prevented pipelines from being published when plugin artifact versions were not specified
+
+- :cask-issue:`CDAP-12284` - Fixed a packaging bug that caused debian packages to include the wrong cdap-data-pipeline and cdap-data-streams artifacts for spark2.
+
+- :cask-issue:`CDAP-12351` - Fixes an issue where truncating a file set did not preserve its base directory's ownership and permissions. 
+
+- :cask-issue:`CDAP-12360` - Fixed an issue where certain excessive logging could cause a deadlock in CDAP master.
+
+- :cask-issue:`CDAP-12371` - In order to execute Hive queries using MR execution engine in CM 5.12 cluster, the 'yarn.app.mapreduce.am.staging-dir' property needs to be set to '/user' in the yarn configuration safety value in CM.
+
+- :cask-issue:`CDAP-12408` - Updates the Create Microservice wizard to allow creating microservices with inbound queues and outbound queues of different types.
```

Comment:
Remove this. Since microservices were added in 4.3.0 itself, this is not a bugfix/enhancement from a previous release.